Client/game.js

```diff
@@ -482,6 +484,36 @@ socket.on('take completed stories', function (stories, numStages, commStrokes =
     $('#ending-scroll').html(scrollHtml);
 });
 
+function likeImage(imageId) {
+	let im = document.getElementById(imageId);
+	if (im.src.includes('like_off.png')) {
+		im.src = 'like_on.png';
+		document.getElementById(imageId.replace('like', 'counter')).style.color = 'green';
```

Comment:
Rather than changing the style in JS, you should add a class in the CSS (eg .liked) and add/remove the class in JS. That keeps it more abstract and also makes it easier to modify the style later